core/src/player.rs

```diff
@@ -1994,6 +2003,12 @@ impl PlayerBuilder {
         self
     }
 
+    /// Sets the root SWF URL provided to ActionScript.
+    pub fn with_spoofed_url(mut self, url: Option<String>) -> Self {
```

Comment:
I think it would be nicer to omit `Option` from here, and call `with_spoofed_url` from `desktop/src/main.rs` only when `--spoof-url` is actually specified. Something like:
```rs
pub fn with_spoofed_url(mut self, url: String) -> Self {
    self.spoofed_url = Some(url);
    self
}

// desktop/src/main.rs
if let Some(url) = opt.spoof_url {
    builder = builder.with_spoofed_url(url);
}
```

Reply:
I think it's better to allow the spoofed url to be explicitly unset by a consumer crate.